tests/homepage.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { test, expect } from '@playwright/test';
+
+test.describe('specific viewport block', () => {
+  test.use({ viewport: { width: 1600, height: 1200 } });
+
+  test('has title', async ({ page }) => {
+    await page.goto('http://localhost:8080');
+
+    // Expect a title "to contain" a substring.
+    await expect(page).toHaveTitle('Agora | Explore Alzheimer\'s Disease Genes');
+  });
+
+  test('has header', async ({ page }) => {
+    await page.goto('http://localhost:8080');
+
+    // Expect header to be visible
+    await expect(page.locator('div#header')).toBeVisible();
+  });
+
+  test('has heading', async ({ page }) => {
+    await page.goto('http://localhost:8080');
+
+    // Expect header to be visible
+    await expect(page.locator('h1')).toBeVisible();
+  });
+
+  test('has footer', async ({ page }) => {
+    await page.goto('http://localhost:8080');
+
+    // Expect header to be visible
+    await expect(page.locator('#footer')).toBeVisible();
+  });
+
+  test('has news', async ({ page }) => {
+    await page.goto('http://localhost:8080');
+
+    // Hamburger menu should be hidden for the given viewport above
+    expect(page.locator('button.header-nav-toggle')).toBeHidden();
+    
+    // look for news link and click it
+    await page.locator('a span', { hasText: 'News' }).click();
```

Comment:
Using the `locator` directly works great, but could also consider using the [Playwright recommended built-in locator](https://playwright.dev/docs/locators#quick-guide): 

```suggestion
    // look for news link and click it
    await page.getByRole('link', { hasText: 'News' }).click();
```

Reply:
I guess this is a mind shift since I wanted to be explicit about the elements used.  I guess the idea is to approach testing with accessibility in mind.  From the docs: "We recommend using text locators to find non interactive elements like div, span, p, etc. For interactive elements like button, a, input, etc. use [role locators](https://playwright.dev/docs/locators#locate-by-role)".

I ended up using the getByRole and name attributes to match the documentation.  I wasn't sure why Playwright  uses 'name' instead of something like 'text' in the docs since html elements can have a 'name' attribute and that could be more confusing.